cloudinit/config/cc_lxd.py

```diff
@@ -73,6 +74,84 @@
                 domain: lxd
             """
         ),
+        dedent(
+            """\
+            # For more complex non-iteractive LXD configuration of networks,
+            # storage_pools, profiles, projects, clusters and core config,
+            # `lxd:preseed` config can be passed directly to the command:
+            #  lxd init --preseed
+            # See https://linuxcontainers.org/lxd/docs/master/preseed/ or
+            # run: lxd init --dump to see viable preseed YAML allowed.
+            #
+            # Preseed settings configuring the LXD daemon for HTTPS connections
+            # on 192.168.1.1 port 9999, a nested profile which allows for
+            # LXD nesting on containers and a limited project allowing for
+            # RBAC approach when defining behavior for sub projects.
+            lxd:
```

Comment:
This example has a few different errors. Can we apply a diff like this?
```diff
diff --git a/cloudinit/config/cc_lxd.py b/cloudinit/config/cc_lxd.py
index a8c137d34..d48045229 100644
--- a/cloudinit/config/cc_lxd.py
+++ b/cloudinit/config/cc_lxd.py
@@ -91,65 +91,65 @@ meta: MetaSchema = {
               preseed:
                 config:
                   core.https_address: 192.168.1.1:9999
-                  preseed:
                 networks:
-                - config:
-                    ipv4.address: 10.42.42.1/24
-                    ipv4.nat: true
-                    ipv6.address: fd42:4242:4242:4242::1/64
-                    ipv6.nat: true
-                  description: ""
-                  name: lxdbr0
-                  type: bridge
-                  project: default
-                  storage_pools:
+                  - config:
+                      ipv4.address: 10.42.42.1/24
+                      ipv4.nat: true
+                      ipv6.address: fd42:4242:4242:4242::1/64
+                      ipv6.nat: true
+                    description: ""
+                    name: lxdbr0
+                    type: bridge
+                    project: default
+                storage_pools:
                   - config:
                       size: 5GiB
                       source: /var/snap/lxd/common/lxd/disks/default.img
                     description: ""
                     name: default
-                    driver: {}
+                    driver: zfs
                 profiles:
-                - config: {}
-                  description: Default LXD profile
-                  devices:
-                    eth0:
-                      name: eth0
-                      network: lxdbr0
-                      type: nic
-                    root:
-                      path: /
-                      pool: default
-                      type: disk
-                  name: default
-                - config: {security.nesting: true}
-                  devices:
-                    eth0:
-                      name: eth0
-                      network: lxdbr0
-                      type: nic
-                    root:
-                      eath: /
-                      pool: default
-                      type: disk
-                  name: nested
+                  - config: {}
+                    description: Default LXD profile
+                    devices:
+                      eth0:
+                        name: eth0
+                        network: lxdbr0
+                        type: nic
+                      root:
+                        path: /
+                        pool: default
+                        type: disk
+                    name: default
+                  - config: {}
+                    security.nesting: true
+                    devices:
+                      eth0:
+                        name: eth0
+                        network: lxdbr0
+                        type: nic
+                      root:
+                        path: /
+                        pool: default
+                        type: disk
+                    name: nested
                 projects:
-                - config:
-                    features.images: true
-                    features.networks: true
-                    features.profiles: true
-                    features.storage.buckets: true
-                    features.storage.volumes: true
-                  description: Default LXD project
-                  name: default
-                - config:
-                    features.images: false
-                    features.networks: true
-                    features.profiles: false
-                    features.storage.buckets: false
-                    features.storage.volumes: false
-                  description: Limited Access LXD project
-                  name: limited
+                  - config:
+                      features.images: true
+                      features.networks: true
+                      features.profiles: true
+                      features.storage.volumes: true
+                    description: Default LXD project
+                    name: default
+                  - config:
+                      features.images: false
+                      features.networks: true
+                      features.profiles: false
+                      features.storage.volumes: false
+                    description: Limited Access LXD project
+                    name: limited
+
+
             """
         ),
     ],
```

Reply:
Thanks. Applied!